Example/RealmResultsController-iOSTests/RealmLoggerTests.swift

```diff
@@ -127,16 +137,57 @@ class RealmLoggerSpec: QuickSpec {
         }
         
         describe("finishRealmTransaction()") {
-            let newObject = RealmChange(type: Task.self, action: .Create, mirror: nil)
-            beforeEach {
-                logger.cleanAll()
-                logger.temporary.append(newObject)
-                logger.cleanAll()
+            let observer = NotificationObserver()
+            var newObject: RealmChange!
+            context("object without mirror") {
+                beforeEach {
+                    newObject = RealmChange(type: Task.self, action: .Create, mirror: nil)
+                    logger.cleanAll()
+                    logger.temporary.append(newObject)
+                    logger.finishRealmTransaction()
+                    logger.cleanAll()
+                }
+                it("Should remove all the objects in each temporary array") {
```

Comment:
Can we please please use a common pattern for tests descriptions?
I like using lowercase and avoid the "should" which I find redundant, you can write `it does this` `it saves that`.